back/spec/factories/followers.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :follower do
+    user
+    followable { create(:project) }
```

Comment:
```suggestion
    association :followable, factory: :project
```
-> it uses the same build strategy for the association.